roles/confluent.test/molecule/rbac-mds-plain-custom-rhel/molecule.yml

```diff
@@ -45,6 +45,19 @@ platforms:
     privileged: true
     networks:
       - name: confluent
+  - name: mds-kafka-broker2
```

Comment:
Wait why did having a single node MDS cause the verification to fail?

Reply:
because the verification was looking for two nodes when it validated the properties.  All other scenarios use two mds nodes, so it made sense to add the second node to both scenarios.

Reply:
why not fix the verify.yml then? I think its better to have scenarios that are different to test more deployments

Reply:
Because all other scenarios are using multiple MDS, so to be consistent across scenarios I added the additional MDS.  It also looks like the original intention was to test with multiple MDS, as the verify was looking for multiple MDS entries.